app/models/spree/order_decorator.rb

```diff
@@ -45,8 +48,22 @@ def commit_avatax_invoice(doc_type)
             :discounted => true,
             :item_code => 'SHIPPING'
         )
+
         invoice_lines << invoice_line
 
+        charge_line = Avalara::Request::Line.new(
+            :line_no => (line_count + 1).to_s,
+            :destination_code => '1',
+            :origin_code => '1',
+            :qty => 1,
+            :amount => charge.to_s,
+            :tax_code => 'FR',
```

Comment:
what does this tax code mean? maybe you should use a constant, or at least add a comment.